.github/scripts/on-push.sh

```diff
@@ -109,7 +109,7 @@ else
     replace_script+="data['packages']['toolchain-xtensa-esp32']['optional']=True;"
     replace_script+="data['packages']['toolchain-xtensa-esp32s3']['optional']=False;"
     replace_script+="data['packages']['tool-esptoolpy']['owner']='tasmota';"
-    replace_script+="data['packages']['tool-esptoolpy']['version']='https://github.com/tasmota/esptool/releases/download/v3.2.1/esptool-3.2.1.zip';"
+    replace_script+="data['packages']['tool-esptoolpy']['version']='https://github.com/tasmota/esptool/releases/download/v3.3/esptool-3.3.zip';"
```

Comment:
questions is: Should we not do this earlier? So all chips use that same version?